requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/PennyLaneAI/pennylane.git@master
+git+https://github.com/PennyLaneAI/pennylane.git@ad/facade-wrapper
```

Comment:
Update the tag after merging https://github.com/PennyLaneAI/pennylane/pull/6046
```suggestion
git+https://github.com/PennyLaneAI/pennylane.git@master
```

Reply:
What do we need to merge #6046?